src/psyclone/dynamo0p3.py

```diff
@@ -4075,15 +4438,24 @@ def __init__(self, call, parent_call):
                     self._unique_fs_names.append(function_space.mangled_name)
                     self._unique_fss.append(function_space)
 
-    def get_arg_on_space(self, func_space):
+    def get_arg_on_space(self, func_space, mangled=True):
```

Comment:
document args

Reply:
In documenting this I realised that I had written this so that the `func_space` argument could be one of two different types. That seemed a bad decision so I have now broken-out the functionality previously obtained by setting `mangled=False` into a new, `get_arg_on_space_name()` routine. I've documented the interfaces for both routines.